test/ctr.bats

```diff
@@ -1021,3 +1021,65 @@ function check_oci_annotation() {
 
 	run ! crictl run "$newconfig" "$TESTDATA"/sandbox_config.json
 }
+
+@test "ctr log linking" {
+	create_runtime_with_allowed_annotation logs io.kubernetes.cri-o.LinkLogs
```

Comment:
I think you should skip this test for kata (when RUNTIME_TYPE=vm) - all tests using "create_runtime_with_allowed_annotation" are failing, probably because of a mismatch between the conf file we generate in this function, and the env variables that we set for kata.